desktop/core/src/desktop/js/reactComponents/Toolbar/ToolbarButton/ToolbarButton.scss

```diff
@@ -0,0 +1,59 @@
+@use "sass:color";
+
+@import '../ToolbarShared.scss';
+
+$antd-btn-padding-horizontal: 12px;
+$focus-border-width: 3px;
+$focus-border-color: $fluid-blue-200;
+
+@mixin tab-focus-apparence() {
+  // Prevent antd transition when modifying padding to allow
+  // for the focus-visible border.
+  transition: 0s padding ease-out;
+
+  &:focus-visible {
+    border-color: $focus-border-color;
+    border-radius: $fluid-border-radius;
+    transition: none;
+  }
+}
+
+.antd {
+  .hue-toolbar-button__wrapper {
+    list-style-type: none;
+    height: 100%;
+    display: flex;
+    align-items: center;
+  }
+
+  .hue-toolbar-button {
+    @include tab-focus-apparence;
+
+    height: var(--toolbar-height);
+    padding: 0 $antd-btn-padding-horizontal;
+    border-width: $focus-border-width;
+    border-color: transparent;
+    border-radius: 0;
+
+    &:hover {
+      background-color: color.adjust($toolbar-background-color, $lightness: -3%);
+    }
+
+    span {
+      font-size: $font-size-base;
+      display: flex;
+      align-items: center;
+      gap: $fluid-spacing-xxs;
+    }
+
+    svg {
+      height: 16px;
```

Comment:
Shouldn't these be relative sizes given that you can vary the height of the toolbar and height: 100% above?

Reply:
Good question. The dynamic toolbar height was added in order to allow taller toolbars (not smaller) and for those cases the content of the buttons (label and icon) were meant to stay the same size. After discussion with Alkarim we concluded that we might not even need the taller toolbars, but I left the option in there since it was already built. But that was a bit sloppy.  A better solution is to align the toolbar with the cuix button sizes, so I have replaced the `height` prop with a `size`  prop that accepts small | medium | large and renders the toolbar in 32, 40 or 48 pixels height with the button icons and labels in the ToolbarButton matching the corresponding cuix button sizes.